src/dodal/beamlines/i19_2.py

```diff
@@ -81,6 +82,15 @@ def synchrotron() -> Synchrotron:
     return Synchrotron()
 
 
+@device_factory()
+def pincol() -> PinholeCollimatorControl:
```

Comment:
Nit: I would call this something more descriptive, in a world of autocomplete being descriptive is more important than being concise. e.g. `pinhole_and_collimator`